apps/www/components/Auth/SignOut.js

```diff
@@ -42,7 +43,8 @@ class SignOut extends Component {
                   // re-load after sign in
                   // - clear apollo cache
                   // - clears potentially authenticated ws conntection
-                  window.location.reload()
+                  //window.location.reload()
+                  router.reload()
```

Comment:
why change this? seems to do 100% the same thing:
https://github.com/vercel/next.js/blob/d67baa0d1fef8a411679c9b9bdea39cb476ce2d5/packages/next/shared/lib/router/router.ts#L879-L881
https://nextjs.org/docs/api-reference/next/router#routerreload